src/framework/components/render/component.js

```diff
@@ -136,10 +131,13 @@ class RenderComponent extends Component {
     _material;
 
     /**
-     * @type {EntityReference}
+     * A reference to the entity to be used as the root bone for any skinned meshes that
+     * are rendered by this component.
+     *
+     * @type {import('../../entity.js').Entity|null}
```

Comment:
Do this instead:

https://github.com/playcanvas/engine/blob/9af3f9efa2f0a45d24368fea680dba9408cd7eed/src/extras/exporters/gltf-exporter.js#L25-L27

Reply:
In other words, once at the top (after regular imports) and then you can use the `Entity` type throughout the module.